src/mrack/run.py

```diff
@@ -198,7 +198,7 @@ def handle(*args, **kwargs):
             ProviderError,
             ApplicationError,
         ) as known_error:
-            logger.error(known_error)
+            logger.error(known_error, extra={"name_override": func.__name__})
```

Comment:
In the end this was necessary. Can't we leave the logger configuration on `src/mrack/__init__.py`, so if any module don't need to set a context can just use the logger with no surprises? The reasoning is that we could call logger from anywhere, without worrying about it breaking in production.

Reply:
Hmmm this should not be needed.

Reply:
I would use only mrack logging so we do not mix up the loggers and the log is consistent.